Process.h

```diff
@@ -113,6 +114,7 @@ typedef struct Process_ {
 
 typedef struct ProcessFieldData_ {
    const char* name;
+   uint8_t pidColumn;
```

Comment:
`#include <stdbool.h>` and use `bool` for this column. Place before/**after** `flags` field. Mixing this in after the name causes quite some noise with the data definitions where this is initialized.